docker/hosts.dnsmasq

```diff
@@ -0,0 +1,12 @@
+# IPv4
+127.0.0.1 leader.mesos
+127.0.0.1 master.mesos
+127.0.0.1 agent.mesos
+127.0.0.1 slave.mesos
+
+# IPv6 - even though it is not used, we still need to define it so that dnsmasq
```

Comment:
ipv6 dns queries are done first by glibc IIRC and this causes confusing error messages (which are safe to ignore BTW).